keyboards/takashicompany/minidivide_max/info.json

```diff
@@ -0,0 +1,544 @@
+                        8,
+                        0
+                    ],
+                    "x": 7.14,
+                    "y": 2.94
+                },
+                {
+                    "matrix": [
+                        8,
+                        1
+                    ],
+                    "x": 7.98,
+                    "y": 2.73
+                },
+                {
+                    "matrix": [
+                        8,
+                        2
+                    ],
+                    "x": 8.82,
+                    "y": 2.52
+                },
+                {
+                    "matrix": [
+                        8,
+                        3
+                    ],
+                    "x": 9.66,
+                    "y": 2.73
+                },
+                {
+                    "matrix": [
+                        8,
+                        4
+                    ],
+                    "x": 10.5,
+                    "y": 2.94
+                },
+                {
+                    "matrix": [
+                        8,
+                        5
+                    ],
+                    "x": 11.34,
+                    "y": 2.94
+                },
+                {
+                    "matrix": [
+                        4,
+                        2
+                    ],
+                    "x": 2.52,
+                    "y": 3.99
+                },
+                {
+                    "matrix": [
+                        4,
+                        3
+                    ],
+                    "x": 3.36,
+                    "y": 3.99
+                },
+                {
+                    "matrix": [
+                        4,
+                        4
+                    ],
+                    "x": 4.2,
+                    "y": 3.99
+                },
+                {
+                    "matrix": [
+                        4,
+                        5
+                    ],
+                    "x": 5.04,
+                    "y": 3.99
+                },
+                {
+                    "matrix": [
+                        9,
+                        0
+                    ],
+                    "x": 6.3,
+                    "y": -2.36
+                },
+                {
+                    "matrix": [
+                        9,
+                        1
+                    ],
+                    "x": 7.14,
+                    "y": -2.36
+                },
+                {
+                    "matrix": [
+                        9,
+                        2
+                    ],
+                    "x": 7.98,
+                    "y": -2.36
+                },
+                {
+                    "matrix": [
+                        9,
+                        3
+                    ],
+                    "x": 8.82,
+                    "y": -2.36
+                }
+            ]
+        }
+    }
+}
```

Comment:
1. adjust x and y positions to match physical board layout - QMK Configurator and `qmk info` have no capability for key rotation
2. add End of File
```suggestion
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0.4},
                {"matrix": [0, 1], "x": 1, "y": 0.4},
                {"matrix": [0, 2], "x": 2, "y": 0.2},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0.2},
                {"matrix": [0, 5], "x": 5, "y": 0.4},
                {"matrix": [5, 0], "x": 9, "y": 0.4},
                {"matrix": [5, 1], "x": 10, "y": 0.2},
                {"matrix": [5, 2], "x": 11, "y": 0},
                {"matrix": [5, 3], "x": 12, "y": 0.2},
                {"matrix": [5, 4], "x": 13, "y": 0.4},
                {"matrix": [5, 5], "x": 14, "y": 0.4},

                {"matrix": [1, 0], "x": 0, "y": 1.4},
                {"matrix": [1, 1], "x": 1, "y": 1.4},
                {"matrix": [1, 2], "x": 2, "y": 1.2},
                {"matrix": [1, 3], "x": 3, "y": 1},
                {"matrix": [1, 4], "x": 4, "y": 1.2},
                {"matrix": [1, 5], "x": 5, "y": 1.4},
                {"matrix": [6, 0], "x": 9, "y": 1.4},
                {"matrix": [6, 1], "x": 10, "y": 1.2},
                {"matrix": [6, 2], "x": 11, "y": 1},
                {"matrix": [6, 3], "x": 12, "y": 1.2},
                {"matrix": [6, 4], "x": 13, "y": 1.4},
                {"matrix": [6, 5], "x": 14, "y": 1.4},

                {"matrix": [2, 0], "x": 0, "y": 2.4},
                {"matrix": [2, 1], "x": 1, "y": 2.4},
                {"matrix": [2, 2], "x": 2, "y": 2.2},
                {"matrix": [2, 3], "x": 3, "y": 2},
                {"matrix": [2, 4], "x": 4, "y": 2.2},
                {"matrix": [2, 5], "x": 5, "y": 2.4},
                {"matrix": [7, 0], "x": 9, "y": 2.4},
                {"matrix": [7, 1], "x": 10, "y": 2.2},
                {"matrix": [7, 2], "x": 11, "y": 2},
                {"matrix": [7, 3], "x": 12, "y": 2.2},
                {"matrix": [7, 4], "x": 13, "y": 2.4},
                {"matrix": [7, 5], "x": 14, "y": 2.4},

                {"matrix": [3, 0], "x": 0, "y": 3.4},
                {"matrix": [3, 1], "x": 1, "y": 3.4},
                {"matrix": [3, 2], "x": 2, "y": 3.2},
                {"matrix": [3, 3], "x": 3, "y": 3},
                {"matrix": [3, 4], "x": 4, "y": 3.2},
                {"matrix": [3, 5], "x": 5, "y": 3.4},
                {"matrix": [8, 0], "x": 9, "y": 3.4},
                {"matrix": [8, 1], "x": 10, "y": 3.2},
                {"matrix": [8, 2], "x": 11, "y": 3},
                {"matrix": [8, 3], "x": 12, "y": 3.2},
                {"matrix": [8, 4], "x": 13, "y": 3.4},
                {"matrix": [8, 5], "x": 14, "y": 3.4},

                {"matrix": [4, 2], "x": 3, "y": 4.1},
                {"matrix": [4, 3], "x": 4, "y": 4.3},
                {"matrix": [4, 4], "x": 5, "y": 4.5},
                {"matrix": [4, 5], "x": 6, "y": 4.7},

                {"matrix": [9, 0], "x": 8, "y": 4.7},
                {"matrix": [9, 1], "x": 9, "y": 4.5},
                {"matrix": [9, 2], "x": 10, "y": 4.3},
                {"matrix": [9, 3], "x": 11, "y": 4.1}
            ]
        }
    }
}

```